content/200-concepts/150-data-platform/08-data-proxy.mdx

```diff
@@ -26,15 +26,15 @@ Please note that the Prisma Data Proxy is an early-stage product and should not
 <Admonition type="warning">
 
 **Not supported** <br />
```

Comment:
I think we could turn this super prominent warning admonition into a "Limitations" headline further down. That would make it much less scary, but still correct. I think our error messages are good enough (from tomorrow) to not having to have this up front and top here.

Agree @albertoperdomo?

Reply:
Agree, I'll change this later in the week.